Cargo.lock

```diff
@@ -708,11 +708,11 @@ dependencies = [
  "http",
  "http-body",
  "hyper",
- "hyper-rustls",
+ "hyper-rustls 0.23.2",
  "hyper-tls",
  "lazy_static",
  "pin-project-lite",
- "rustls",
+ "rustls 0.20.8",
  "tokio",
  "tower",
```

Comment:
Is it possible to avoid these duplicated dependencies? 🥲

Reply:
Let me take a look.

Reply:
`aws-smithy-client` depends on `hyper-rustls 0.23` directly...

https://docs.rs/crate/aws-smithy-client/0.55.3/source/Cargo.toml

Reply:
Get it. Sorry that I misread it. I thought it's `opendal` but it's `aws-smithy-client` 😇